reference/catalog.v3.yml

```diff
@@ -1424,8 +1430,39 @@ paths:
               instance:
                 type: string
             title: Not Found
+        '422':
+          description: |-
+            Unprocessable Entity. 
+
+            May occur if the `Content-Type` header is set to `multipart/form-data` rather than `application/json` when creating a product image using `image_url`.
+          schema:
+            description: ''
+            type: object
+            properties:
+              status:
+                type: number
+              title:
+                type: string
+                minLength: 1
+              type:
+                type: string
+                minLength: 1
+            required:
+              - status
+              - title
+              - type
+            x-examples:
+              example-1:
+                status: 422
+                title: image_url must be present if uploading by url
+                type: 'https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes'
+          examples:
```

Comment:
Just wondering if the example below is necessary since we have the same example above.